### DISCUSSION TITLE
Help with code

### DISCUSSION THREAD

**Discussion by ro2kell** [community/community#103090] [Programming Help]
### Select Topic Area

Question

### Body

Hi, I'm hoping you can help me with a code from GitHub. I saw someone demonstrate how their resume builder works and he left a GitHub link to the code. I just wanted to get the code so I can use it on my website. I don't know anything about coding and its terminologies but how do I get the code and paste it on my WordPress website? Here is the link to the CV builder. https://github.com/faizan2515/CV-Generator

**Comment by hubertle43100** [comment]
Hello `ro2kell`,

Here are the steps I would take to solve your issues

`1. Download Code:`

- Go to the GitHub repository: [CV-Generator](https://github.com/faizan2515/CV-Generator).
- Click on "Code" and choose "Download ZIP."
- Extract the ZIP file on your computer.

`2. Review Code:`
- Open the extracted folder, and check for any instructions or documentation.

`3. WordPress Integration:`
- Seek help from a developer to integrate the code into your WordPress website.

`4. Hosting:`
- Make sure your hosting platform supports custom code.

`5. Professional Assistance:`
- If you're unsure, consider hiring a developer for assistance.


### Hope that helps ⭐
